components/datadog/operator/component.go

```diff
@@ -0,0 +1,27 @@
+package operator
+
+import (
+	compkubernetes "github.com/DataDog/test-infra-definitions/components/kubernetes"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+
+	"github.com/DataDog/test-infra-definitions/components"
+)
+
+// OperatorOutput is used to import the Operator component
+type OperatorOutput struct { // nolint:revive, We want to keep the name as <Component>Output
+	components.JSONImporter
+
+	Operator compkubernetes.KubernetesObjRefOutput `pulumi:"operator"`
```

Comment:
<!-- Suggestions propose improvements to the current subject. It’s important to be explicit and clear on what is being suggested and why it is an improvement. -->
Should use `json` decorator on outputs
**💬 suggestion**
```suggestion
	Operator compkubernetes.KubernetesObjRefOutput `json:"operator"`
```